src/components/Match/matchColumns.jsx

```diff
@@ -365,61 +377,77 @@ export const performanceColumns = [
     tooltip: strings.tooltip_lane_efficiency,
     field: 'lane_efficiency',
     sortFn: true,
-    displayFn: (row, col, field) => (field ? field.toFixed(2) : '-'),
+    displayFn: (row, col, field) => (field ? `${(field * 100).toFixed(2)}%` : '-'),
+    relativeBars: {
```

Comment:
probably this should be treated as a relative to max, efficiency can go over 100 (e.g. alchemist)